emails/sensitive-changes-email.tsx

```diff
@@ -0,0 +1,83 @@
+import { DISCORD_URL } from "@/pages/about";
+import { Text, Link } from "@react-email/components";
+import * as React from "react";
+import CTABlock from "./ui/cta-block";
+import Base from "./ui/base";
+import List from "./ui/list";
+import { CreateMemberFields } from "@/lib/firebase-helpers/members";
+import Layout from "./ui/layout";
+import Logo from "./ui/logo";
+import Code from "@/components/Code";
+
+export default function SensitiveChangesEmail({
+  name = "[Name Inoa]",
+  changes = `{"link": {"old": "https://old.url", "new": "https://new.url"}}`,
```

Comment:
Hm, both these fields are required with Typescript. You cannot use the `SensitiveChangesEmail`  component without those props.

This also makes it so we can work on the template and see changes with the filler copy. Otherwise we'd be working blind, and we'd default to an impossible-to-achieve layout. Feels risky for anyone who runs `dev:email`.